resources/services/teams/teams.go

```diff
@@ -370,7 +371,7 @@ func Teams() *schema.Table {
 					{
 						Name:        "team_id",
 						Description: "The id of the team",
-						Type:        schema.TypeBigInt,
+						Type:        schema.TypeString,
```

Comment:
@roneli I don't know whether this should be `TypeBigInt` or `TypeString` -- the generator generates String, so I went with that for now.

Reply:
Oh nevermind, the tests failed. Guess it should be TypeBigInt after all